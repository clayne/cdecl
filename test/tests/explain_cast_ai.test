cdecl @ @ @ explain (auto int)x @ 65
