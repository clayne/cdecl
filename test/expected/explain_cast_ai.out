explain (auto int)x
              ^
15: error: can not cast into auto
